src/components/file-upload/index.md.njk

```diff
@@ -23,6 +23,54 @@ There are 2 ways to use the file upload component. You can use HTML or, if you
 
 {{ example({group: "components", item: "file-upload", example: "default", html: true, nunjucks: true, open: false, id: "default-2"}) }}
 
+### Error messages
+
+Check that a valid date is entered by the user. Invalid dates should be reported to the user like this:
+
+{{ example({group: "components", item: "file-upload", example: "error", html: true, nunjucks: true, open: false, size: "m"}) }}
+
+Make sure errors follow the guidance in [error message](/components/error-message/) and have specific error messages for specific error states.
+
+#### If nothing has been uploaded
+Say ‘Select a [whatever they need to upload]’.<br>
+For example, ‘Select a report’.
+
+#### If the file is the wrong file type
+Say‘The [whatever] must be a [list of file types]’.<br>
+For example, ‘The report must be a CSV or ODS’ or ‘The upload must be a JPG, BMP, PNG, TIF or PDF’.
+
+#### If the file is too big
+Say ‘The [whatever] must be smaller than [largest file size]’.<br>
+For example, ‘The CSV must be smaller than 2MB’.
+
+#### If the file is empty
+Say ‘The [whatever] is empty – select a [whatever] with data in it’.<br>
+For example, ‘The CSV is empty – select a CSV with data in it’.
```

Comment:
We're alternating between the concept of 'uploading' and the concept of 'selecting' – can we try and stick to one?

Reply:
Also not sure that 'a CSV with data in it' is a good description of what's expected.

Reply:
I think this is because of the different terminology used in the two different services. I will stick to upload because that is what is in the standard example.

Reply:
The empty file error is odd but I think 'a CSV with data in it' is all we can say without any more context. Do you think 'The [whatever] is empty' without the extra information is better?

Reply:
I agree it is specific enough. Going with 'The selected file is empty' to be consistent.